Automattic/jetpack
# VideoPress: Add inspector controls

#### Changes proposed in this Pull Request:

* Adds the remaining controls to the sidebar - except Poster and Video File Settings
* Only show sidebar controls once the video is present in the block

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


NA

#### Testing instructions:






* Add a new VP
* Add a video
* Check the new settings on the sidebar only appears after the video is added to the block

![Captura de tela de 2022-06-28 15-21-37](https://user-images.githubusercontent.com/971483/176254910-63c30a63-07c6-4b7d-af76-b2f4bd1635c2.png)

Make sure they all work properly



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202518480088382
